lib/transformations/extractTextPlugin/extractTextPlugin.js

```diff
@@ -1,32 +1,36 @@
+// @flow
+import type { Ijscodeshit, IPath, INewExpression, IObjectExpression, IMemberExpression, ILiteral } from '../../types'; // eslint-disable-line node/no-unsupported-features
 const utils = require('../utils');
 
-function findInvocation(j, node, pluginName) {
-	return j(node)
+function findInvocation(j: Ijscodeshit, node: IPath<IMemberExpression>, pluginName: string): boolean {
+	let found: boolean = false;
```

Comment:
BTW Any reason to introduce the variable here?

Reply:
I guess it was to acknowledge flow in a explicitly way of the type that was returning. I think I remove it